tests/utils/test_process_utils.py

```diff
@@ -136,23 +136,21 @@ def test_should_kill_process(self):
         num_process = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().count("\n")
         assert before_num_process == num_process
 
-    @pytest.mark.quarantined
     def test_should_force_kill_process(self):
-        before_num_process = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().count("\n")
 
         process = multiprocessing.Process(target=my_sleep_subprocess_with_signals, args=())
         process.start()
         sleep(0)
 
-        num_process = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().count("\n")
-        assert before_num_process + 1 == num_process
+        all_processes = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().splitlines()
+        assert str(process.pid) in map(lambda x: x.strip(), all_processes)
```

Comment:
Would this work?

```suggestion
        all_processes = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().splitlines(keepends=False)
        assert str(process.pid) in all_processes
```

Reply:
That would not work. The returned values do not have EOL. They are prefixed with spaces: `                1` for example.

Reply:
Crap. Thanks for the explanation.